<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>873</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QLCDNumber{
	border-color: rgb(13, 13, 13);
	color: rgb(6, 6, 6);
}</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QLabel" name="label_temp_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;PibotoLt&quot;;</string>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_date">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(18, 18, 18);
font: 75 italic 12pt &quot;PibotoLt&quot;;</string>
   </property>
   <property name="digitCount">
    <number>17</number>
   </property>
   <property name="mode">
    <enum>QLCDNumber::Hex</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1111</width>
     <height>681</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="push_tempgraph">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>560</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperature Graph</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_humgraph">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>560</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Humidity Graph</string>
    </property>
   </widget>
   <widget class="matplot_widget" name="matplot_widget" native="true">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>59</y>
      <width>521</width>
      <height>471</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>480</width>
      <height>320</height>
     </size>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text_statusline">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>620</y>
      <width>1011</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(4, 4, 4);
color: rgb(12, 255, 0);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_stop">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>10</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-color: rgb(255, 38, 10);</string>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_alert">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>530</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>451</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="push_th_t">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>99</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="push_th_h">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>100</y>
       <width>99</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="text_alarm_t">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="text_alarm_h">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>50</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>TEMPERATURE</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>HUMIDITY</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>451</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(231, 231, 231);
border-color: rgb(8, 8, 8);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLCDNumber" name="lcd_timestamp">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(92, 191, 200);
background-color: rgb(255, 255, 255);
font: 75 italic 12pt &quot;PibotoLt&quot;;</string>
     </property>
     <property name="digitCount">
      <number>8</number>
     </property>
     <property name="mode">
      <enum>QLCDNumber::Hex</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_temp">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;PibotoLt&quot;;</string>
     </property>
     <property name="text">
      <string>TEMPERATURE</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcd_humidity">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(92, 191, 200);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="digitCount">
      <number>8</number>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcd_temperature">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(92, 191, 200);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="digitCount">
      <number>8</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_humidity">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>HUMIDITY</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_timestamp">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TIMESTAMP</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_t_unit">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>30</y>
       <width>68</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Celcius</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_h_unit">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>70</y>
       <width>68</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Percent</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="push_refresh">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Sans Serif&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>REFRESH</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_c_f">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>300</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Sans Serif&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>C / F</string>
    </property>
   </widget>
   <zorder>frame_3</zorder>
   <zorder>push_tempgraph</zorder>
   <zorder>push_humgraph</zorder>
   <zorder>matplot_widget</zorder>
   <zorder>text_statusline</zorder>
   <zorder>push_stop</zorder>
   <zorder>text_alert</zorder>
   <zorder>frame</zorder>
   <zorder>push_refresh</zorder>
   <zorder>push_c_f</zorder>
  </widget>
  <zorder>frame_2</zorder>
  <zorder>label_temp_2</zorder>
  <zorder>lcd_date</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>matplot_widget</class>
   <extends>QWidget</extends>
   <header location="global">matplot_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
